### PR TITLE
Deepedit prostate

### PR DESCRIPTION
**Pull request by pritesh-mehta**
Addition of deepedit_prostate app and app screenshot in docs/images/sample-apps

### CODE REVIEW

File: deepedit_prostate/main.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+        self.model_dir = os.path.join(app_dir, "model")
+        self.pretrained_model = os.path.join(self.model_dir, "pretrained.pt")
+        self.final_model = os.path.join(self.model_dir, "model.pt")
+
+        self.download(
+            [
+                (
+                    self.pretrained_model,
+                    "https://github.com/Project-MONAI/MONAILabel/releases/download/data/deepedit_prostate.pt",
```
Comment (SachidanandAlle) [inline deepedit_prostate/main.py:37]: This link not exists

Comment (pritesh-mehta) [reply deepedit_prostate/main.py:37]: @diazandr3s can you update the link after you have saved the .pt I sent you?

Comment (diazandr3s) [reply deepedit_prostate/main.py:37]: Thanks! Here is the link

https://github.com/diazandr3s/MONAILabel-Apps/releases/download/pretrained/deepedit_prostate.pt

Comment (diazandr3s) [reply deepedit_prostate/main.py:37]: Please make sure it works as you'd expect

Comment (pritesh-mehta) [reply deepedit_prostate/main.py:37]: download failed - does it need to be renamed pretrained.pt? See trace:

[2021-07-15 15:23:17,676] [INFO] (uvicorn.error) - Started server process [17464]
[2021-07-15 15:23:17,678] [INFO] (uvicorn.error) - Waiting for application startup.
[2021-07-15 15:23:17,679] [INFO] (monailabel.utils.others.app_utils) - Initializing App from: C:\Users\pmeht\Documents\MONAILabel-Apps\deepedit_prostate; studies: C:\Users\pmeht\Documents\monai_label\data\PROSTATEx\imagesTr
[2021-07-15 15:23:18,384] [INFO] (monailabel.interfaces.app) - Downloading resource: C:\Users\pmeht\Documents\MONAILabel-Apps\deepedit_prostate\model\pretrained.pt from https://github.com/Project-MONAI/MONAILabel/releases/download/data/deepedit_prostate.pt
pretrained.pt: 0.00B [00:01, ?B/s]
Download failed from https://github.com/Project-MONAI/MONAILabel/releases/download/data/deepedit_prostate.pt to C:\Users\pmeht\AppData\Local\Temp\tmpobastpdg\pretrained.pt.

Comment (diazandr3s) [reply deepedit_prostate/main.py:37]: I think you copy the wrong link. Please copy exactly this one: https://github.com/diazandr3s/MONAILabel-Apps/releases/download/pretrained/deepedit_prostate.pt

Comment (pritesh-mehta) [reply deepedit_prostate/main.py:37]: Yes, you are correct. Now works. A model folder is created with pretrained.pt within it.

**Pull request merged**